picopass/picopass.c

```diff
@@ -97,6 +97,27 @@ Picopass* picopass_alloc() {
     view_dispatcher_add_view(
         picopass->view_dispatcher, PicopassViewLoclass, loclass_get_view(picopass->loclass));
 
+    picopass->plugin_manager =
```

Comment:
I guess you are planning to add more plugins in future?

Reply:
No other plugin ideas right now, but I liked the folder form of handling this vs the single plugin form (based on the example apps)